docs/openj9_support.md

```diff
@@ -117,9 +213,3 @@ When public support for an operating system version ends, OpenJ9 can no longer b
 | Linux on IBM Z 64-bit       | Ubuntu 16.04             | gcc 4.8.5                       |                |
```

Comment:
This line in the JDK9 environment shows the kernel level in brackets, which I don't think is required.